dotnet/samples/KernelSyntaxExamples/Example71_AssistantDelegation.cs

```diff
@@ -36,26 +36,25 @@ public static async Task RunAsync()
         var plugin = KernelPluginFactory.CreateFromObject<MenuPlugin>();
 
         var menuAssistant =
-            await AssistantBuilder.FromDefinitionAsync(
-                TestConfiguration.OpenAI.ApiKey,
-                model: OpenAIFunctionEnabledModel,
-                template: EmbeddedResource.Read("Assistants.ToolAssistant.yaml"),
-                new[] { plugin });
+            await new AssistantBuilder()
+                .WithOpenAIChatCompletion(OpenAIFunctionEnabledModel, TestConfiguration.OpenAI.ApiKey)
+                .FromTemplate(EmbeddedResource.Read("Assistants.ToolAssistant.yaml"))
+                .WithDescription("Answer questions about the menu use the tool.")
```

Comment:
A couple of typos here.  Maybe you meant something like:

```suggestion
                .WithDescription("Answer questions about how the menu uses the tool.")
```